### PR TITLE
Wp-env: Add MySQL port info to start logs

### PR DESCRIPTION
**Pull request by jeyip**
## Description

Follow up for #21545. See this comment https://github.com/WordPress/gutenberg/pull/21545#issuecomment-614349354.

Screen Shot 2021-01-25 at 3 57 38 PM

## How has this been tested?

- Apply this PR
- Run `npx wp-env start`
- Take note of the output logs. Copy the MySQL port.
- Download [Sequel Ace](https://github.com/Sequel-Ace/Sequel-Ace) I went through the macOS app store.
- Open the application, select "TCP/IP" connection, and input the following info:
  ```
  Name: WordPress Gutenberg
  Host: 127.0.0.1
  Database: wordpress
  Port: [PASTE_MY_SQL_PORT_HERE]
  Time Zone: Use Server Time Zone
  ``` 
- Click on connect
- Connection should be successful
- Once connected, you should see all database tables for your local WordPress instance like so:
Screen Shot 2021-01-22 at 2 06 44 PM 

## Types of changes

New feature

## Checklist:
- [ ] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

File: packages/env/lib/commands/start.js [hunk 1]
```diff
@@ -182,9 +182,20 @@ module.exports = async function start( { spinner, debug, update, xdebug } ) {
...
 	);
+	const mySQLPort = mySQLAddress.stdout.split( ':' ).pop();
+
+	spinner.text = 'Done!'
+		.concat( '\n' )
+		.concat( `MySQL is listening on port ${ mySQLPort }` )
+		.concat( 'WordPress started' )
+		.concat( siteUrl ? ` at ${ siteUrl }` : '.' )
+		.concat( '\n' )
+		.concat( 'WordPress E2E testing site started' )
```
Comment (noahtallen) [inline packages/env/lib/commands/start.js:197]:
```suggestion
		.concat( 'WordPress test site started' )
```

File: packages/env/lib/commands/start.js [hunk 2]
```diff
@@ -182,9 +182,20 @@ module.exports = async function start( { spinner, debug, update, xdebug } ) {
...
+	const e2eSiteUrl = config.env.tests.config.WP_TESTS_DOMAIN;
+	const mySQLAddress = await exec(
+		`docker-compose -f ${ dockerComposeConfigPath } port mysql 3306`
 	);
+	const mySQLPort = mySQLAddress.stdout.split( ':' ).pop();
+
+	spinner.text = 'Done!'
+		.concat( '\n' )
+		.concat( `MySQL is listening on port ${ mySQLPort }` )
+		.concat( 'WordPress started' )
```
Comment (noahtallen) [inline packages/env/lib/commands/start.js:194]:
```suggestion
		.concat( 'WordPress development site started' )
```

**Review by noahtallen — COMMENTED** [review]
Nice, thank you for adding this!

I think it makes sense to make the messages more consistent, and I'm also wondering if we can change this:

```
~/gutenberg $ npx wp-env start                                                                                                                                         
✔ Done!
MySQL is listening on port 55002
WordPress started at http://localhost:8888/
WordPress E2E testing site started at http://localhost:8889/ (in 1s 443ms)
```

To something more like:

```
~/gutenberg $ npx wp-env start                                                                                                                                         
MySQL is listening on port 55002
WordPress started at http://localhost:8888/
WordPress E2E testing site started at http://localhost:8889/

✔ Done! (in 1s 443ms)
```

Moving the "done" message to the end, right by the timer.

**Comment by jeyip** [comment]
@noahtallen Thanks for the review! I agree with your suggestions. I'll implement them ASAP.

**Comment by jeyip** [comment]
Updated output:
Screen Shot 2021-01-25 at 3 57 38 PM

**Review by noahtallen — APPROVED** [review]
Let's do it!

**Comment by noahtallen** [comment]
Actually, could we order the SQL port at the end instead of at the beginning?

**Comment by jeyip** [comment]
Done 👍 

Screen Shot 2021-01-25 at 5 01 27 PM

**Comment by jeyip** [comment]
Pinging @noisysocks @ocean90 -- any additional thoughts on this PR? I'm planning to merge it soon.

**Pull request merged**